src/components/account/EditDetails.js

```diff
@@ -5,8 +5,9 @@ import TextField from '@mui/material/TextField';
 import DialogContent from '@mui/material/DialogContent';
 import './style.css';
 
-const EditDetails = ({ user }) => {
+const EditDetails = ({ user, handleSubmit, handleChange }) => {
   const { email } = user;
+  const reqBodyEmail = email
```

Comment:
Same here, you can just use the new `email` const you've created rather than doubling up on variables